modules/new_profile/help/new_profile.md

```diff
@@ -1,8 +1,9 @@
 # New Profile
 
-By clicking on "New Profile" under the Candidate Menu, users with appropriate permissions can register a new candidate for their study site. Once inside the "New Profile" page, the "Date of Birth" field must be entered twice, in order to minimize error in data entry. 
-The PSCID refers to an alphanumeric identifier entered by the user. This identifier is typically comprised of a site-specific code (e.g., AAA), followed by a numeric code specific to the candidate at that site (e.g., AAA0000). Other fields are customized for specific projects and can be configured as drop-down fields or user input fields. 
-Once all the required data fields are completed, the user can click on the "Create" button to finish registering the candidate. It is crucial that no mistakes in data entry are made at this point, as information cannot be modified after clicking the "Create" button. 
+This is where—with the appropriate permissions—you can register a new candidate for your study site. 
```

Comment:
```suggestion
This is where you can register a new candidate for your study site. 
```

I don't think this help text is visible unless the user already has the correct permissions so this aside isn't needed.